pilot/pkg/model/service.go

```diff
@@ -566,7 +566,49 @@ func (s *ServiceAttributes) DeepCopy() ServiceAttributes {
 	// Nested mutexes are configured to be ignored by copystructure.Copy.
 	// Disabling `go vet` warning since this is actually safe in this case.
 	// nolint: vet
-	return copyInternal(*s).(ServiceAttributes)
+	out := *s
```

Comment:
remove the above comment

Reply:
followed up in https://github.com/istio/istio/pull/40990